Xamarin.Essentials/Contacts/Contacts.macos.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Contacts;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Contacts
+    {
+        static Task<Contact> PlatformPickContactAsync() => throw ExceptionUtils.NotSupportedOrImplementedException;
```

Comment:
Maybe this file should be merged with ios? It is just the picker that macos does not have, but the rest of the read is using the same APIs. And probably the write.

We can `#if __IOS__` the picker code...